docs/release-notes.md

```diff
@@ -7,6 +7,42 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## **1.5.0** for Magento Commerce 2.4.1
+
+-  {:.new}<!--Issue 510, 511, 512, 513-->Immersive, full-screen editing. Editing Page Builder content is now full-screen only for all areas controlled by Page Builder. This includes CMS pages, Product and Category pages, Blocks, and Dynamic Blocks. Full-screen editing puts the focus on your content and provides a view that better matches the user experience on the storefront.
+
+-  {:.new}<!--Issue 544-->Page Builder content previews. By default, Page Builder now provides content previews not only for CMS pages and Blocks, but for Product and Category pages as well. You can configure this feature to be on or off for Product and Category pages using the new Page Builder Content Preview setting, accessed in Stores > Configuration > Content Management > Advanced Content Tools.
```

Comment:
since you distinguished between "Blocks" & "Dynamic Blocks" above, you should probably change `By default, Page Builder now provides content previews not only for CMS pages and Blocks` to `By default, Page Builder now provides content previews not only for CMS pages, Blocks, and Dynamic Blocks,`

Reply:
Check. I tried to get away with using blocks generically in this case, but I should have probably lower-cased it. However, since you noticed it, I'll fixed it to your specs. 🥇